ezyrb/ann.py

```diff
@@ -78,7 +120,6 @@ def predict(self, new_point):
         :return: the predicted values via the ANN.
         :rtype: numpy.ndarray
         """
-        new_point = np.array(new_point)
-        new_point = torch.from_numpy(new_point).float()
-        y_new=self.model(new_point)
-        return y_new.detach().numpy()
+        new_point = self._convert_numpy_to_torch(new_point)
```

Comment:
In order to perform successfully the predict method in case of a list is given as the input new_point (this is, for instance, the case of the example in the tutoral), I suggest to replace this line by "new_point = self._convert_numpy_to_torch(np.array(new_point))".